src/Http/Controllers/Configuration/AccessController.php

```diff
@@ -108,6 +108,20 @@ public function editRole($role_id)
             ));
     }
 
+    public function renameRole($role_id)
+    {
+        $this->validate(request(), [
```

Comment:
As this is a POST request, please use a request validator (injected as `Request $request` instead of `$role_id`).

Reply:
Accidentally resolved?